### DISCUSSION TITLE
Add ability to ignore tests at runtime

### DISCUSSION THREAD

**Discussion by yanganto** [crablang/crab#96] [Ideas]
This is an issue to be ignored in Rust for the long term.
https://github.com/rust-lang/rust/issues/68007

Also, there is already a PR for this.
https://github.com/rust-lang/rust/pull/96574

Could we consider adding it in Crab lang? :crab:

**Comment by VasilStoianov** [comment]
I am new here but I think i can give it a try if it's ok for you guys ?

**Comment by yanganto** [comment]
It is happy to see this and also looks after the following PRs.  But I do not really know if the Crab is ready to make any difference between Rust.
